src/Layouts/HomeLayout.js

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
+import { connect } from "react-redux";
 import UsersTableComponent from "../components/UsersTableComponent";
+import { usersServices } from '../services/usersServices';
 
-export default class HomeLayout extends Component {
+class HomeLayout extends Component {
+    componentDidMount() {
+        this.props.dispatch(usersServices());
```

Comment:
service layer contains all http request related to user, so It just not returning single action. I think you need to refactor it first @azukhrufy

Reply:
Ok, i'll try to fix it